src/components/DatePicker/Years.jsx

```diff
@@ -92,7 +96,8 @@ Years.propTypes = {
   displayStart: PropTypes.number,
   displayEnd: PropTypes.number,
   displayStatus: PropTypes.func,
-  onChange: PropTypes.func
+  onChange: PropTypes.func,
+  label: PropTypes.string
```

Comment:
Maybe we will not use this only to pass strings, I think PropTypes.node is a better option

Reply:
6693dd4c5757ae26b356ee7422623c258ee968e5